horizomer/misc/taxonomy/tree_by_taxdump.py

```diff
@@ -0,0 +1,142 @@
+# this script generates a tree of a set genomes based on their NCBI taxonomy
+# (taxdump) annotations
+# usage: python me.py nodes.dmp genome2taxid.tsv output.nwk
```

Comment:
Reasonable for a first pass.  But I'd suggest using either click or Argparse, so that a help menu is available.